app/decode/morse.js

```diff
@@ -81,3 +87,14 @@ export function decode(message) {
 
   return parts
 }
+
+export function rearrange(message, newChars) {
+  let rearrangedMessage = new String(message)
+  rearrangedMessage = rearrangedMessage.replaceAll('-', 'x')
```

Comment:
Let's use method as second parameter to replaceAll: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#specifying_a_function_as_the_replacement